src/main/java/LacksDataBaseProject/AddAndRemoveLack.java

```diff
@@ -0,0 +1,37 @@
+package LacksDataBaseProject;
+
+import java.util.Scanner;
+
+public class AddAndRemoveLack {
+    static Scanner input = new Scanner(System.in);
+
+    public static void addlack(User user){
+        System.out.println("Item: ");
+        //Item item
+        System.out.println("Required amount: ");
+        int requiredAmount = input.nextInt();
+        System.out.println("Unit of measure: ");
+        String unitOfMeasure = input.next();
+        System.out.println("Comment: ");
+        String forwarderAdditionalComment = input.next();
+        // Lack lack = new Lack(item, requiredAmount, unitOfMeasure, forwarderAdditionalComment,....
+        // i dodatkowe dane które przekazuje Purchaser
```

Comment:
Tutaj w sumie nie dane, które przekazuje Purchaser, tylko metoda createMissingLackData, bo to ona wypełnia wszystkie brakujące pola... tak zeby juz w momencie wypelniania danych przez Forwardera powstal Lack, a Purchaser dopiero później modyfikuje wybrane pola.